### PR TITLE
Feat/create customer creation page

### PR DESCRIPTION
**Pull request by mathieurobert1**
I created the customer creation page.
image

### CODE REVIEW

**Review by theophile-jr — DISMISSED** [review]

File: angular-frontend/src/app/pages/customers-list/customers-list.component.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                        {{ customer.name }} {{ customer.surname }}
+                    </td>
+                    <td>{{ customer.email }}</td>
+                    <td>{{ customer.birth_date }}</td>
+                    <td></td>
+                    <!-- <td *ngIf="this._auth.isManager()" ><img *ngIf="this.payments[customer.id][0].payment_method == 'PayPal'" src="assets/paypal-card.jpg" class="image-card-payment">
+                        <img *ngIf="this.payments[customer.id][0].payment_method == 'Credit Card'" src="assets/cb-card.jpg" class="image-card-payment">
+                        <img *ngIf="this.payments[customer.id][0].payment_method == 'Bank Transfer'" src="assets/wire-transfer.jpg" class="image-card-payment">
+                        <span *ngIf="this.payments[customer.id][0].payment_method != 'PayPal' && this.payments[customer.id][0].payment_method != 'Credit Card' && this.payments[customer.id][0].payment_method != 'Bank Transfer'">{{ this.payments[customer.id][0].payment_method }}</span>
+                    </td> -->
```
Comment (Njord201) [inline angular-frontend/src/app/pages/customers-list/customers-list.component.html:85]: Remove the comment.

File: angular-frontend/src/app/pages/customers-list/customers-list.component.ts [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+            this.customers = await this.employeesService.getCustomers(this.employee?.id);
+        } else {
+            this.customers = await this.customersService.getCustomers();
+        }
+        this.allCustomers = this.customers;
+        // if (this._auth.isManager()) {
+        //     for (let customer of this.customers) {
+        //         this.payments[customer.id] = await this.paymentService.getPaymentsCustomer(customer.id);
+        //     }
+        // }
```
Comment (Njord201) [inline angular-frontend/src/app/pages/customers-list/customers-list.component.ts:47]: Remove the comment.

File: angular-frontend/src/app/pages/register-customer/register-customer.component.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+                <option value="Leo">Leo</option>
+                <option value="Virgo">Virgo</option>
+                <option value="Libra">Libra</option>
+                <option value="Scorpio">Scorpio</option>
+                <option value="Sagittarius">Sagittarius</option>
+                <option value="Capricorn">Capricorn</option>
+                <option value="Aquarius">Aquarius</option>
+                <option value="Pisces">Pisces</option>
+            </select>
+        </div>
```
Comment (Njord201) [inline angular-frontend/src/app/pages/register-customer/register-customer.component.html:29]: We can't define the astrological sign automatically ?

**Review by Njord201 — CHANGES_REQUESTED** [review]

File: angular-frontend/src/app/pages/customers-list/customers-list.component.html (hunk 1 above)
Comment (mathieurobert1) [reply angular-frontend/src/app/pages/customers-list/customers-list.component.html:85]: It's to wait the new Axel's feature about method payments.

File: angular-frontend/src/app/pages/customers-list/customers-list.component.ts (hunk 2 above)
Comment (mathieurobert1) [reply angular-frontend/src/app/pages/customers-list/customers-list.component.ts:47]: It's to wait the new Axel's feature about method payments.

File: angular-frontend/src/app/pages/register-customer/register-customer.component.html (hunk 3 above)
Comment (mathieurobert1) [reply angular-frontend/src/app/pages/register-customer/register-customer.component.html:29]: If you have an idea to compute signs based on birth date, give me your ideas

**Review by theophile-jr — APPROVED** [review]

**Review by Njord201 — CHANGES_REQUESTED** [review]
Check my comment.

**Review by theophile-jr — APPROVED** [review]

**Review by Njord201 — APPROVED** [review]

**Pull request merged**